threadspy/threads_api.py

```diff
@@ -487,39 +488,278 @@ def __toggle_auth__post_request(self, url: str):
             token = self.token
         if token is None:
             raise "Token not found"
-        headers = self.__get_default_headers()
-        res = self.http_client.post(url, headers=headers)
-        return res
+        headers = self.__get_app_headers()
+        response = self.http_client.post(url, headers=headers)
+        return response
 
     def like(self, post_id: str) -> bool:
-        user_id = self.get_current_user_id()
-        res = self.__toggle_auth__post_request(
-            f"{BASE_API_URL}/api/v1/media/{post_id}_${user_id}/like/"
+        """
+        like a post.
+
+        Args:
+            post_id (str): post identifier
+
+        Returns:
+            boolean and if verbose mode is enabled, prints response dict
+        """
+        user_id = self.user_id or self.get_current_user_id()
+        response = self.__toggle_auth__post_request(
+            url=f"{BASE_API_URL}/api/v1/media/{post_id}_{user_id}/like/",
         )
-        return res.json()["status"] == "ok"
+        if self.verbose:
+            print("[LIKE]", response.json())
+        return response.json()["status"] == "ok"
```

Comment:
```suggestion
        data = response.json()
        if self.verbose:
            print("[LIKE]", data)
        return data["status"] == "ok"
```
Can we store the `response.json()` into another variable? (Same for other usages, too)

Reply:
Sure, I guess we should change it in any other methods as well

Reply:
yup

Reply:
can you change them?

Reply:
I'm going to cover that in my next pull request, as well as the issue with the publishing URL. =)

Reply:
omg thanks 🙏

Reply:
No need to thank me, I'm using the library too xD